io.sourceforge.qoobar.Qoobar.yml

```diff
@@ -0,0 +1,78 @@
+app-id: io.sourceforge.qoobar.Qoobar
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: qoobar
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --device=dri
+  - --filesystem=host
```

Comment:
why is host access needed? isn't `xdg-music` enough?

Reply:
I'm not sure. I wanted to grant access to f.e. download dir and some other places. Maybe `home`?

Reply:
there is `xdg-download` too. you can specify more than one location.